src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/AssessmentEntity.kt

```diff
@@ -37,7 +37,7 @@ data class AssessmentEntity(
 
   @ManyToOne
   @JoinColumn(name = "allocated_to_user_id")
-  val allocatedToUserId: UserEntity,
+  val allocatedToUser: UserEntity,
```

Comment:
No migration needed, the database column name renames the same - just changed this as it's actually an entire User once JPA has done its magic rather than just the ID